### ISSUE TITLE
Slash command description is read in Inline Chat but not in Panel

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#186409]
Testing #186218

* Type `/` command in Inline Chat box, NVDA reads both the command and its description
* Type `/` command in Panel Chat, NVDA reads only command id but not its description

**Comment by meganrogge** [comment]
@TylerLeonhardt can you still repro on windows?

on mac, it reads both - though I notice there's no space separator. 
https://github.com/microsoft/vscode/assets/29464607/8c5f2cc8-8ffb-4eda-a330-0067256f3175

**Comment by meganrogge** [comment]
it's also not read in the inline chat for me
https://github.com/microsoft/vscode/assets/29464607/00084149-2620-40a2-8e91-903bced58f84

**Comment by meganrogge** [comment]
Ok - I see that this is by design here since there is text, but in this case, it's just a slash command. Perhaps slash command text should be an exception? @aiday-mar 

https://github.com/microsoft/vscode/blob/ad395686d24694709c16942a2ae5f36b90ad384a/src/vs/workbench/contrib/inlineChat/browser/inlineChatWidget.ts#L422

**Comment by aiday-mar** [comment]
Agreed the slasj command should be read out loud since it applies an action

**Comment by roblourens** [comment]
Is this about reading the suggest widget? Or alerting once you select something from the suggest widget? I think there was another issue about this where we talked about how that might read it twice but I can't remember where that was.

**Comment by meganrogge** [comment]
This is about alerting once something has been selected. We should have the same behavior in both places.

**Comment by roblourens** [comment]
For just reading the suggest widget or selecting an item from the suggest widget, I see the same behavior for all suggest widgets, in panel chat, inline chat, and the normal editor. Here's what I see

- Activate the suggest widget
  - If the suggest details flyout is shown, then it reads the command name but not the description
  - If the flyout is collapsed so the description shows inline in the row, then it reads the name and description
- Select the suggest item
  - It reads something like "Selection replaced" and doesn't read details of what was inserted

For the first case, I'm not sure that I can do anything to customize it, but I'd guess that there is some way to focus the details flyout for screenreader users so that it gets read?

For the second, we could trigger an alert manually via a command when the item is selected, but the editor doesn't do that, so I don't know whether it's expected or not.

lmk what you think @meganrogge

**Comment by meganrogge** [comment]
There is no way to focus the details flyout ATM, but there has been ongoing discussion about this and a plan to address that. https://github.com/microsoft/vscode/pull/190096

> If the flyout is collapsed so the description shows inline in the row, then it reads the name and description

Is there a way to make it always collapsed?

I would avoid using `alert` here if we can.

**Comment by meganrogge** [comment]
This is a duplicate of https://github.com/microsoft/vscode/issues/167900
